config/locales/custom/admin.yml

```diff
@@ -209,6 +212,10 @@ en:
         information_texts_menu:
           budgets: "Budgets"
     site_customization:
+      content_blocks:
+        content_block:
+          names:
+            homepage_main_content: After Financed Domains
```

Comment:
Considering the Financed Domains component isn't rendered anymore, I'm not sure whether this makes sense :thinking:. How about "homepage"?